tools/shared.py

```diff
@@ -38,8 +38,8 @@ class FatalError(Exception):
   pass
 
 
-def exit_with_error(*args):
-  logging.error(*args)
+def exit_with_error(msg, *args):
```

Comment:
Is the explicit `msg` here to make it clearer that the first argument is special, or to better avoid calling this improperly (with 0 arguments)?
Either way, nice.

Reply:
This was part of a separate change to actually re-raise in debug mode.. but its seemed like an innocuous thing to squeeze in there.